### PR TITLE
Pr add mysql

### PR DESCRIPTION
**Pull request by ChillerDragon**
There is no actual MySQL code yet just the dependency.
``cmake .. -DMYSQL=ON``
compiled without warnings on my debian machine.

### CODE REVIEW

**Comment by Learath2** [comment]
What's the point if there is no mysql code? :D

**Comment by ChillerDragon** [comment]
@Learath2 you can see it as a WIP I will try to also get the mysql code running. But in case I rage quit someone else can pick it up and does not have to fight with dependencys again.

**Comment by ChillerDragon** [comment]
screenshot_2019-12-07_15-30-15

``/map``, ``/rank`` and ``/top5`` work. I could not test ``/points`` because i did not even get that running with ddnet6.

``/save`` is not yet supported.

**Comment by ChillerDragon** [comment]
This replaces teeworlds-libs and the other folder using the download.py script with ddnet-libs submodule. So make sure to get the submodule if you have cloned the repository already:

```
git submodule update --init --recursive
```

File: src/game/server/teams.cpp [hunk 1]
```diff
@@ -442,7 +442,7 @@ void CGameTeams::OnTeamFinish(CPlayer** Players, unsigned int Size, float Time,
 	bool CallSaveScore = false;
 
 #if defined(CONF_SQL)
-	CallSaveScore = g_Config.m_SvUseSQL;
+	// CallSaveScore = g_Config.m_SvUseSQL; // TODO: SQL
```
Comment (def-) [inline src/game/server/teams.cpp:445]: ?

**Pull request merged**